xml/deployment_installation.xml

```diff
@@ -899,10 +897,11 @@ network-interfaces: |
 #cloud-config
 debug: True
 disable_root: False
+ssh_deletekeys: False
```

Comment:
Care to elaborate?

Reply:
No -- I was taking the example files from Bugzilla. 

Googling suggests that "for public instances this should definitely be 'true'". Happy to change it if you think that is preferable.